submissions/iakow/friends-app/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Friends</title>
+  </head>
+
+  <body class="app-container">
+    <header class="header">
+      <h1>My Imaginary Friends</h1>
+    </header>
+
+    <aside class="filters-block" id="filters-block">
+      <div class="filter-name">
+        <input
+          class="filter-name__input filter"
+          id="filter-name"
+          type="search"
+          placeholder="Name"
+        />
+      </div>
+
+      <div class="filter-age">
+        <input
+          class="filter-age__input filter"
+          id="filter-age-min"
+          type="number"
+          min="0"
+        />
+        <span class="filter-age__divider"></span>
+        <input
+          class="filter-age__input filter"
+          id="filter-age-max"
+          type="number"
+          min="0"
+        />
+      </div>
+
+      <div class="filter-sex">
+        <button class="filter-sex__input filter button" id="filter-sex"></button>
+      </div>
+
+      <div class="sort-age">
+        <button class="sort-age__input filter button" id="sort-age"></button>
+      </div>
+
+      <div class="sort-name">
+        <button class="sort-name__input filter button" id="sort-name"></button>
+      </div>
```

Comment:
The fact is that a bunch of radio buttons do not fit into the layout on a mobile resolution) So I decided to make buttons. But now I realized that it would be easier to work with radio buttons in the script (no need to manually change the input value). Now I'm thinking about how I can combine radio buttons for desktop resolution and buttons for mobile ...

Reply:
@Iakow 

> radio buttons do not fit into the layout on a mobile resolution 

This is not problem of radio buttons, this is problem with your layout)
Seems like i found bug: when i click on button image (arrow or sex icon ) blinks and dont render properly fix this please

It is very bad  UX practice in this case to use button instead of radio